_data/projects/trapteam-cc.yml

```diff
@@ -0,0 +1,14 @@
+name: *Trap Team CC*
```

Comment:
@steviebeenz Please remove "*" to make your code pass travis build

Reply:
Please remove "*" for everywhere

Reply:
Just fixed, thank you for the suggestion!